frontend/src/pages/onboarding/FavoriteGymsForm.tsx

```diff
@@ -8,14 +8,15 @@ import { getGymList } from 'src/services/gyms';
 import useSafeRequest from 'src/hooks/services/useSafeRequest';
 import { useSnackbar } from 'notistack';
 import RHFAutoMultiSelect from 'src/components/hook-form/RHFAutoMultiSelect';
-import { CacheKey } from 'src/config';
+import { CacheKey, USEREQUEST_OPTIONS_STALE_TIME } from 'src/config';
 
 export const FavoriteGymsForm = () => {
   const { formState } = useFormContext<OnboardingFormValues>();
   const { errors } = formState;
   const { enqueueSnackbar } = useSnackbar();
   const { data: gyms } = useSafeRequest(getGymList, {
```

Comment:
Could consider abstracting this to a hook so other parts can use.

Reply:
I was thinking about that too. will do in future PR